test/cfg/bsd.c

```diff
@@ -32,10 +32,13 @@ void nullPointer_setlinebuf(FILE *stream)
 // #9323, #9331
 void verify_timercmp(struct timeval t)
 {
+    // cppcheck-suppress duplicateExpression
     (void)timercmp(&t, &t, <);
+    // cppcheck-suppress duplicateExpression
     (void)timercmp(&t, &t, <=);
     (void)timercmp(&t, &t, ==);
     (void)timercmp(&t, &t, !=);
+    // cppcheck-suppress duplicateExpression
     (void)timercmp(&t, &t, >=);
     (void)timercmp(&t, &t, >);
```

Comment:
It's odd that we don't warn here

Reply:
We run into some weird logic in `combineOperators()`, which causes `.` tokens to have differen `originalName()`:
`if (Token::Match(t, "( & %name% )") && !Token::simpleMatch(t->previous(), ">")) {`